server/internal/usecase/interactor/item.go

```diff
@@ -41,13 +41,19 @@ func (i Item) FindByID(ctx context.Context, itemID id.ItemID, operator *usecase.
 }
 
 func (i Item) FindBySchema(ctx context.Context, schemaID id.SchemaID, p *usecasex.Pagination, operator *usecase.Operator) (item.List, *usecasex.PageInfo, error) {
-	_, err := i.repos.Schema.FindByID(ctx, schemaID)
+	s, err := i.repos.Schema.FindByID(ctx, schemaID)
 	if err != nil {
 		return nil, nil, err
 	}
+	sfl := s.Fields()
+	sfids := id.FieldIDList{}
+	for _, sf := range sfl {
+		sfids.Add(sf.ID())
+	}
```

Comment:
Move line 44~52 inside Run2 func

Also it is hard to read, so

```go
	sfl := s.Fields()
	sfids := id.FieldIDList{}
	for _, sf := range sfl {
		sfids.Add(sf.ID())
	}
```

can be changed to

```go
	sfids := s.Fields().IDs()
```

If `IDs` method is not defined at `schema.FieldList`, define it.